media-api/app/lib/ImageResponse.scala

```diff
@@ -260,7 +260,7 @@ class ImageResponse(config: MediaApiConfig, s3Client: S3Client, usageQuota: Usag
     val source = uri.toURL.toExternalForm
     val signed = new String(Base64.encodeBase64URLSafe(source.getBytes), "UTF-8")
     val resizingUrl = Seq(config.imgopsUri(instance),  "no-signature",
-      "auto_rotate:false", "strip_metadata:true", "strip_color_profile:true",
+      "auto_rotate:true", "strip_metadata:true", "strip_color_profile:true",
```

Comment:
Comment; needs to be conditional to avoid breaking cropping of existing images with unused Exif orientation.